jenkinsfiles/dev

```diff
@@ -142,17 +146,16 @@ pipeline {
         stage('Publish images') {
             steps {
                 withDockerRegistry([credentialsId: "docker-hub-credentials", url: ""]) {
-                    script {
-                        // Docker images are built in a matrix of different base images
-                        // all these images are used for testing but only one is published and endorsed for use in production
-                        // which is the DOCKER_IMAGE_NAME_PUBLISH_SOURCE
-                        sh "docker tag ${DOCKER_IMAGE_NAME_PUBLISH_SOURCE} ${DOCKER_IMAGE_NAME_PUBLISH_TARGET}"
-                        sh "docker push ${DOCKER_IMAGE_NAME_PUBLISH_TARGET}"
```

Comment:
We previously pushed images tagged as master and latest which we do not anymore if I understand the changes correctly. If this is ok you will need to adapt https://github.com/dhis2/operations-handbook/pull/6/files

Reply:
Maybe I should add that I do agree with this change 😋 @radnov and I just opted for keeping the repo/tag structure as it is when working on jib. We thought this might need its own discussion/PR which is not related to this work.